spec/01-unit/08-router_spec.lua

```diff
@@ -3484,6 +3484,27 @@ for _, flavor in ipairs({ "traditional", "traditional_compatible", "expressions"
         assert.is_nil(match_t.matches.headers)
       end)
 
+      it("uri_captures works well with the optional capture group. Fix #13014", function()
```

Comment:
I think that it is not a proper test case, It did not re-produce the situation in #13014.
The test uri should be `1984/profile`, and `match_t.matches.uri_captures[1]` should be `nil`

Perhaps we should add another test case for it.

Reply:
@chronolaw OK, I will add another test case after the previous question is confirmed.